src/include/homestore/homestore.hpp

```diff
@@ -95,6 +95,31 @@ struct HS_SERVICE {
     }
 };
 
+struct HOMESTORE_FLIPS {
```

Comment:
please move it from homestore.cpp to btree code, these flips are btree specific, you can add a function which sets all this flips .

Reply:
done